### PR TITLE
[#21804] Docs: Update limitations for read committed isolation

### PR DESCRIPTION
**Pull request by pkj415**

### CODE REVIEW

File: docs/content/stable/architecture/transactions/read-committed.md [hunk 1]
```diff
@@ -1565,7 +1565,10 @@ Read Committed interacts with the following feature:
 
     For more details, see [#12494](https://github.com/yugabyte/yugabyte-db/issues/12494).
 
-* Read restart and serialization errors are not internally handled in read committed isolation if the query's response size exceeds the YB-TServer GFlag `ysql_output_buffer_size`, which has a default value of 256KB (see [#11572](https://github.com/yugabyte/yugabyte-db/issues/11572)).
+* Read restart and serialization errors are not internally handled in read committed isolation:
+- if the query's response size exceeds the YB-TServer GFlag `ysql_output_buffer_size`, which has a default value of 256KB (see [#11572](https://github.com/yugabyte/yugabyte-db/issues/11572)).
```
Comment (foucher) [inline docs/content/stable/architecture/transactions/read-committed.md:1569]: The sub-bullets aren't formatting the way you might want them to. At least in the preview.

File: docs/content/stable/architecture/transactions/read-committed.md [hunk 2]
```diff
@@ -1565,7 +1565,10 @@ Read Committed interacts with the following feature:
...
+- if multiple semi-colon separated statements in a single query string are sent via the simple query protocol (see [#21833](https://github.com/yugabyte/yugabyte-db/issues/21833)).
```
Comment (foucher) [inline docs/content/stable/architecture/transactions/read-committed.md:1570]: Semicolon is one word

File: docs/content/stable/architecture/transactions/read-committed.md [hunk 3]
```diff
@@ -1565,7 +1565,10 @@ Read Committed interacts with the following feature:
...
+- for statements other than the first one in a batch sent by the driver (see [#21607](https://github.com/yugabyte/yugabyte-db/issues/21607))
```
Comment (foucher) [inline docs/content/stable/architecture/transactions/read-committed.md:1571]: The GHI referenced is slightly off from what the sub-bullet states. I think it should say "except for [GHI] currently" or something like that. The other bugs referenced are for when the bug is fixed the sub-bullet would go away.

File: docs/content/stable/architecture/transactions/read-committed.md (hunk 2 above)
Comment (pkj415) [reply docs/content/stable/architecture/transactions/read-committed.md:1570]: done

File: docs/content/stable/architecture/transactions/read-committed.md (hunk 3 above)
Comment (pkj415) [reply docs/content/stable/architecture/transactions/read-committed.md:1571]: done

**Comment by foucher** [comment]
LGTM

**Pull request merged**